### ISSUE TITLE
Home -> NewProblemsForm: Incorrect Content Order 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#769]
The tabular data which shows the existing problems in a set is positioned before the dialog’s title heading.  Users will not only be confused about the purpose of the table itself, but also not have any way to determine what the dialog is for.  This is an issue compounded by the lack of an effective title for this modal. 
 
Suggested Solution 
 
Move the heading with the dialog’s title so that it appears as the very first element in the body of the modal.

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/e385fb5781a8d7c019fd1e5385ba529669d49425.
